rdl/version.go

```diff
@@ -3,4 +3,4 @@
 
 package rdl
 
-const Version = "1.4.15"
+const Version = "1.4.16"
```

Comment:
Only bump this if you actually are going to make a release, please. That said, we should roll a release :-)